packages/ckeditor5-html-embed/src/htmlembedediting.js

```diff
@@ -143,6 +143,21 @@ export default class HtmlEmbedEditing extends Plugin {
 					domContentWrapper = domElement;
 
 					renderContent( { domElement, editor, state, props } );
+
+					// Since there is a `data-cke-ignore-events` attribute set on the wrapper element in the editable mode,
+					// the explicit `click` handler on the `capture` phase is needed to move the selection onto the whole
+					// HTML embed widget.
+					domContentWrapper.addEventListener( 'click', () => {
```

Comment:
Wondering why couldn't we attach the listener on`textarea#focus`? Shouldn't it be easier?

Reply:
In widgets, it's handled on `mousedown` so maybe we should do the same here? The click event is triggered on a `mouseup` so there is a visible difference in behavior.

Reply:
Done - the `click` event listener has been replaced with `mousedown` event listener.